src/error-handler.ts

```diff
@@ -0,0 +1,40 @@
+import ApplicationError from './exceptions/HttpException';
+import { Request, Response, NextFunction } from 'express';
+import logger from './helper/logger';
+import util from 'util';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export default (err: ApplicationError, req: Request, res: Response, next: NextFunction): void => {
+	// set locals, only providing error in development
+	res.locals.message = err.message;
+	res.locals.error = req.app.get('env') === 'development' ? err : {};
+
+	if (req.app.get('env') === 'development') {
```

Comment:
how are errors logged in production?

Reply:
I fixed this. Errors are generally logged.